### DISCUSSION TITLE
Store failures using incremental model

### DISCUSSION THREAD

**Discussion by olivd4400** [dbt-labs/dbt-core#10558] [Q&A]
Hello,

Is there a way to store failures using an incremental model?

For the time being this does not seem to be supported
`Compilation Error in test primary_key_unique_2 (sources/sources_glue_2.yml)
  'incremental' is not a valid value for `store_failures_as`. Accepted values are: ['ephemeral', 'table', 'view']`

Also the doc states

> A test's results will always replace previous failures for the same test, even if that test results in no failures.

This would be helpful to keep a daily trace of failing records. We are using Glue adapter and it would be nice to keep failing records in an ACID table (Iceberg or Deltalake)

Thanks in advance

**Comment by cecabrera** [comment]
Hi! I am having the same issue. Thanks for the hard work!
